src/ContractInteract/Organization.js

```diff
@@ -0,0 +1,151 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+'use strict';
+
+const Web3 = require('web3');
+
+const AbiBinProvider = require('../AbiBinProvider');
+const Contracts = require('../Contracts');
+const { sendTransaction } = require('../utils/Utils');
+const {
+  validateConfigExists,
+  validateConfigKeyExists,
+} = require('./validation');
+
+const ContractName = 'Organization';
+
+class Organization {
+  /**
+   * Constructor for Organization.
+   *
+   * @param {Object} web3 Web3 object.
+   * @param {string} address Organization contract address.
+   */
+  constructor(web3, address) {
+    if (web3 instanceof Web3) {
+      this.web3 = web3;
+    } else {
+      const err = new TypeError(
+        "Mandatory Parameter 'web3' is missing or invalid",
+      );
+      throw err;
+    }
+
+    if (!Web3.utils.isAddress(address)) {
+      const err = new TypeError(
+        "Mandatory Parameter 'address' is missing or invalid.",
```

Comment:
Generally, it would be nice for these kinds of errors to include the faulty variable in the error so that it is easier to debug for the consumer. Possibly in the whole PR.